satpy/readers/aapp_l1b.py

```diff
@@ -237,11 +238,12 @@ def calibrate(self,
 
         if dataset_id.name in ("3a", "3b") and self._is3b is None:
             # Is it 3a or 3b:
-            is3b = np.expand_dims(
-                np.bitwise_and(
-                    np.right_shift(self._data['scnlinbit'], 0), 1) == 1, 1)
+            is3b = np.expand_dims(np.bitwise_and(self._data['scnlinbit'], 3) == 1, 1)
             self._is3b = np.repeat(is3b,
                                    self._data['hrpt'][0].shape[0], axis=1)
+            is3a = np.expand_dims(np.bitwise_and(self._data['scnlinbit'], 3) == 0, 1)
```

Comment:
I notice that these operations don't seem to be dask at all. As such, could we only do this `bitwise_and` once? Right now, it is used for both 3b and 3a checks. Or, even better, convert this reader to use dask better.

Reply:
Yes, we discussed that here. @mraspaud indicated that this could need a daskification overhaul. I just wanted to make sure we get this improvement in now. The Dask could come in the next step. Also, there is no test for this reader it seems, right?

Reply:
Right. It seems there are no tests.

Reply:
I will see what I might have/take time to fix. Both concerning Dask and tests. But, feel under pressure fixing our transition to our Satpy based production, and this is a minor thing in this regard...

Reply:
Ok, so I will merge this now, given the time constraints we have on this being fixed and the time it would take to overhaul this (tests and dask-friendliness) and we open an issue to remind us on what needs to be done.